README.md

```diff
@@ -1,27 +1,179 @@
 # Court Transcript Pipeline
 
-## Problem Statement:
+---
+
+## Problem Statement
+
 The National Archives release transcripts of real court hearings every day in plain text format. However, the sheer volume and format of these transcripts make them challenging for the average person to consume and analyse effectively. Key questions about court proceedings, such as judge behaviour and case outcomes, remain unanswered due to limited accessibility and resources. Additionally, the decline in journalism leads to the lack of public understanding of court judgments. 
 
-## Elevator Pitch:
+## Elevator Pitch
+
 Our project aims to bridge the gap between court proceedings and public understanding by developing a data pipeline to automate the enhancement, discoverability, and analysis of real courtroom documents. 
 
-## Data Sources:
+## Data Sources
+
 - All Case Law Files: Updates approximately 12 times per day.
 - All Judges in the UK: Updates irregularly.
 
-## Proposed Solution & Functionality:
+## Proposed Solution & Functionality
+
 Our automated data pipeline will:
+
 - Monitor for new court transcripts and process them accordingly.
 - Parse text to extract key information and store it in a structured format.
 - Utilise GPT-4 API for summarising transcripts in various ways, including categorisation, argument summaries, and event detection.
 - Create a searchable dashboard to access and analyse extracted data.
 - Deploy infrastructure using Terraform for scalability and manageability.
 - Include thorough unit tests for critical code components.
 
-## Planned Outputs:
-- Deployed pipeline for automated transcript processing.
-- Deployed dashboard website for user-friendly access to analysed court data.
+## Deliverables:
```

Comment:
Great adding this section, making it known what our product is and what can be done with it.